products/stratos/conf/data-bridge/thrift-client-config.xml

```diff
@@ -0,0 +1,23 @@
+<!--
+  ~ Copyright 2005-2011 WSO2, Inc. (http://wso2.com)
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<!--Fields to be read and parsed into WSO2CEPStatisticsPublisher-->
+<thriftClientConfiguration>
+    <username>test User</username>
```

Comment:
@supunr I'm not entirely sure, however as far as I understand these values (username, password, ip and port) would be set using Puppet or a shell script (for Docker containers) right? If that is the case wouldn't using replaceable strings (ex: USERNAME, PASSWORD, IP, PORT) instead of "test user" be more suitable?